operators/endpointmetrics/controllers/observabilityendpoint/observabilityaddon_controller.go

```diff
@@ -169,7 +179,11 @@ func (r *ObservabilityAddonReconciler) Reconcile(ctx context.Context, req ctrl.R
 		if err != nil {
 			if errors.IsNotFound(err) {
 				log.Error(err, "OCP prometheus service does not exist")
-				util.ReportStatus(ctx, r.Client, obsAddon, "NotSupported")
+				// ACM 8509: Special case for hub/local cluster metrics collection
+				// We do not report status for hub endpoint operator
+				if !hubMetricsCollector {
+					util.ReportStatus(ctx, r.Client, obsAddon, "NotSupported")
```

Comment:
is it better to check whether current deployment is hub metrics collector inside the func ReportStatus()?